### PR TITLE
Adds the routes for the data aggregation pages

### PR DESCRIPTION
**Pull request by andrea-kyurchiev**
### What

Adds the routes to the new data aggregation pages

### How to review
Requires: 
[https://github.com/andrea-kyurchiev/dp-frontend-search-controller/tree/feat/reworks-data-aggregation-pages](https://github.com/andrea-kyurchiev/dp-frontend-search-controller/tree/feat/reworks-data-aggregation-pages)
[https://github.com/andrea-kyurchiev/dp-search-api/tree/feat/apply-date-filters-to-query](https://github.com/andrea-kyurchiev/dp-search-api/tree/feat/apply-date-filters-to-query)
[https://github.com/andrea-kyurchiev/dp-design-system/tree/feat/adds-filters-for-data-aggregation](https://github.com/andrea-kyurchiev/dp-design-system/tree/feat/adds-filters-for-data-aggregation)

### Who can review
Anyone

### CODE REVIEW

**Review by jondewijones — COMMENTED** [review]
Looks good to me but one thing. Not sure if this is in scope but some of these "end points" work across multiple urls. For example `/publications` currently works across multiple taxonomy nodes as well as at the root directory

https://www.ons.gov.uk/publications
https://www.ons.gov.uk/economy/publications
https://www.ons.gov.uk/economy/grossdomesticproductgdp/publications
https://www.ons.gov.uk/businessindustryandtrade/publications
https://www.ons.gov.uk/businessindustryandtrade/business/publications
etc. etc.

**Comment by lindenmckenzie** [comment]
> Looks good to me but one thing. Not sure if this is in scope but some of these "end points" work across multiple urls. For example `/publications` currently works across multiple taxonomy nodes as well as at the root directory
> 
> https://www.ons.gov.uk/publications https://www.ons.gov.uk/economy/publications https://www.ons.gov.uk/economy/grossdomesticproductgdp/publications https://www.ons.gov.uk/businessindustryandtrade/publications https://www.ons.gov.uk/businessindustryandtrade/business/publications etc. etc.

Think this is something I can take away @jondewijones and discuss with Andrea / F&T

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 | GEOGRAPHY_ENABLED                | false                                     | Geography feature toggle                                                                 |
 | FEEDBACK_CONTROLLER_URL          | <http://localhost:25200>                  | The URL of dp-frontend-feedback-controller                                               |
 | SEARCH_CONTROLLER_URL            | <http://localhost:25000>                  | The URL of dp-frontend-search-controller                                                 |
+| NEW_DATA_AGGREGATION_PAGES_ENABLED   | true                                     | Enables the new data aggregation pages                                                             |
```
Comment (lindenmckenzie) [inline README.md:18]: Let's omit the 'NEW' on this - one day it won't be :D

Comment (andrea-kyurchiev) [reply README.md]: Resolved but realistically I would imagine the whole flag would be removed once the transition to the updated pages is made

File: config/config.go [hunk 2]
```diff
@@ -40,9 +40,11 @@ type Config struct {
 	RedirectSecret                  string        `envconfig:"REDIRECT_SECRET" json:"-"`
 	ReleaseCalendarControllerURL    string        `envconfig:"RELEASE_CALENDAR_CONTROLLER_URL"`
 	ReleaseCalendarEnabled          bool          `envconfig:"RELEASE_CALENDAR_ENABLED"`
+	RendererURL                     string        `envconfig:"RENDERER_URL"`
```
Comment (lindenmckenzie) [inline config/config.go:43]: Why have you added this back in?

**Review by lindenmckenzie — APPROVED** [review]

**Comment by lindenmckenzie** [comment]
Closing this PR as have cherrypicked over to a feature branch: https://github.com/ONSdigital/dp-frontend-router/pull/428

**Pull request closed without merging**